step3/src/repository/index.ts

```diff
@@ -0,0 +1,41 @@
+import fs from 'fs';
+import { TodoItem } from '../domain/TodoItem';
+
+class Repository {
```

Comment:
```suggestion
class Repository<T> {
```
이렇게 Generic 타입을 사용하여 유연한 형태로 관리할 수 있도록 작성하면 더 좋을 것 같아요!